block/submit.go

```diff
@@ -126,22 +126,19 @@ func (m *Manager) HandleSubmissionTrigger() error {
 		return fmt.Errorf("create next batch to submit: %w", err)
 	}
 
-	if err := m.ValidateBatch(nextBatch); err != nil {
-		return fmt.Errorf("validate batch: %w", err)
-	}
-
 	resultSubmitToDA, err := m.submitNextBatchToDA(nextBatch)
```

Comment:
why not do here the same you did with the SL? ie calling the `m.DAClient.SubmitBatch` directly?

Reply:
done